Exercise9/Main.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+
+int main()
+{
+	char myString[30] = "A string of thirty characters";
+	char myReversedString[30];
+
+	int size;
+	for (size = 0; myString[size] != '\0'; size++);
+	size;
+
+	for (int i = 0; i < size; i++)
+	{
+		myReversedString[size - i - 1] = myString[i];
+		if (myString[i + 1] == '\0')
```

Comment:
You're checking that you're in the last character of your string to assign the \0 to your `myReversedString[size]`. This will always happen at the end of the loop, so there is no need to actually do it inside the loop. Why not just move the line `myReversedString[size] = \0` to be called after the loop? That way you can eliminate the condition.

Always be mindful of what needs and doesn't need to be inside a for loop.